graph/typeDefs.graphql

```diff
@@ -977,6 +1035,91 @@ type EditCommentResponse implements Response {
   errors: [UserError!]
 }
 
+# UpdateSettingsInput is the input used to input the global site settings. This
+# will override the existing settings, so all fields must be included.
+input UpdateSettingsInput {
```

Comment:
Same here as the other PR, can I submit just a subset of the input and only modify the subset keeping the rest the same?

Reply:
Same as the other one, all fields are required because we can't distinguish in-between a null/undefined answer.